css/portfolio.css

```diff
@@ -128,33 +129,20 @@ nav {
     /*transition: all 1s ease;*/
 /*}*/
 
-.project-hover:hover {
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    background: rgba(220, 0, 0, 0.5);
-    opacity: 0;
+/*.project-hover:hover {*/
```

Comment:
You have commented out this CSS, if you don't need it you should probably delete it

Reply:
actioned